include/fmt/core.h

```diff
@@ -1402,21 +1396,21 @@ FMT_API void vprint(std::FILE *f, wstring_view format_str, wformat_args args);
   **Example**::
 
     fmt::print(stderr, "Don't {}!", "panic");
-  \endrst
- */
-template <typename... Args>
-inline void print(std::FILE *f, string_view format_str, const Args &... args) {
-  format_arg_store<format_context, Args...> as(args...);
-  vprint(f, format_str, as);
-}
-/**
-  Prints formatted data to the file *f* which should be in wide-oriented mode
+
+  **Hint**::
```

Comment:
I don't think `::` is correct here. It's probably better to just have a normal sentence: 

```
Prints formatted data to the file *f*. For wide format strings, *f* should be in wide-oriented mode ...
```

Reply:
Well, if you say so 😄 I have no clue about rst and just copied the syntax from three lines above. Thanks for the correction and the improved wording.